MCprep_addon/materials/generate.py

```diff
@@ -958,6 +958,32 @@ def set_saturation_material(mat):
 		sat_node.mute = not bool(saturate)
 		sat_node.hide = not bool(saturate)
 
+def create_node(tree_nodes, node_type, **kwargs):
+	"""Function to create node"""
+	if node_type == 'ShaderNodeMixRGB': # For MixRGB in 3.4 become Legacy 
+		if util.min_bv((3, 4, 0)):
```

Comment:
`NOTE_FOR_INTERNAL_REFACTORING` (sorta)

There are plans for `MCprepEnv` to check for 2.8/3.0. 3.4 is still debatable as that brought a couple of big changes that make MCprep have to do workarounds, but at the same time it's not as big as say 2.8 or 3.0 were